docs/csharp/fundamentals/types/snippets/classes/Program.cs

```diff
@@ -1,4 +1,6 @@
-﻿public class Person
+using System;
```

Comment:
The using is unnecessary due to the use of `ImplicitUsings`:

https://github.com/dotnet/docs/blob/09d90f573fcf78373a7c18b93938b74f97e6ddf3/docs/csharp/fundamentals/types/snippets/classes/classes.csproj#L7